lib/src/area.dart

```diff
@@ -0,0 +1,130 @@
+import 'dart:math';
+
+import '../helpers.dart';
+import '../meta.dart';
+
+/// Takes one or more features and returns their area in square meters.
+///
+/// ```dart
+/// Feature<Polygon> poly = Feature<Polygon>(
+///   geometry: Polygon(coordinates: [
+///     [
+///       Position(125, -15),
+///       Position(113, -22),
+///       Position(117, -37),
+///       Position(130, -33),
+///       Position(148, -39),
+///       Position(154, -27),
+///       Position(144, -15),
+///       Position(125, -15)
+///     ]
+///   ]),
+/// );
+///
+/// var area = turf.area(polygon);
+/// ```
+num? area(GeoJSONObject geojson) {
+  return geomReduce<num>(geojson, (value, geom, _, __, ___, ____) {
+    return value! + calculateArea(geom!);
+  }, 0);
+}
+
+/// Calculate Area
+num calculateArea(GeometryType geom) {
+  num total = 0;
+  int i;
+  switch (geom.type) {
+    case GeoJSONObjectType.polygon:
+      return polygonArea((geom as Polygon).coordinates);
+    case GeoJSONObjectType.multiPolygon:
+      geom as MultiPolygon;
+      for (i = 0; i < geom.coordinates.length; i++) {
+        total += polygonArea(geom.coordinates[i]);
+      }
+      return total;
+    case GeoJSONObjectType.point:
+    case GeoJSONObjectType.multiPoint:
+    case GeoJSONObjectType.lineString:
+    case GeoJSONObjectType.multiLineString:
+      return 0;
+    case GeoJSONObjectType.geometryCollection:
+      final geometryCollection = geom as GeometryCollection;
+      for (i = 0; i < geometryCollection.geometries.length; i++) {
+        total += calculateArea(geometryCollection.geometries[i]);
+      }
+      return total;
```

Comment:
`GeometryCollection`, `Feature`, or `FeatureCollection` can not be the case, because it is based on the `geomEach`, which already loops over all of the concrete `GeometryTypes`, so you don't need to do it here! 
 https://github.com/dartclub/turf_dart/blob/78f68e98c2f392cc3db71311a4825fb60c553085/lib/src/meta/geom.dart#L144

Reply:
and then let's work with a default case that thows an `Exception`

Reply:
@lukas-h Ok, how would the exception look like?

Reply:
`throw Exception('unsupported type ${geom.type');`

Reply:
@lukas-h Is this ok now?

Reply:
yes, lgtm! 😄